Comp2024/src/main/java/frc/robot/autos/AutoPreloadScore.java

```diff
@@ -62,7 +62,7 @@ public AutoPreloadScore(List<PathPlannerPath> ppPaths, CommandSwerveDrivetrain d
         new LogCommand(getName(), "Drive to spike while intaking"),
         new ParallelCommandGroup(
             drivetrain.getPathCommand(ppPaths.get(1)),
-            new AcquireNote(intake, led, hid).withTimeout(1.5)
+            new AcquireNote(intake, led, hid)

```

Comment:
All of our usages of AcquireNote command in auto routines will need a timeout value. Since they are run in parallel with the path commands (so the intake gets deployed and starts running), that timeout needs to be the duration of the path PLUS some time to grab the note (this is maybe 250 ms + path time). Most of the time we are grabbing notes from spikes, so it is running very short paths of 0.75 sec or less. An acquireNote timeout of 1.0 second should cover all of these cases. However, the first centerline grab will be different, but let's do the math on those separately.

Reply:
fixed